### ISSUE TITLE
Support non-integer shape values for Gamma distribution

### ISSUE THREAD

**Issue by propensive** [propensive/capricious#2]
Currently the `k` value must be an integer, and there's no way to specify a non-integral parameter.

**Comment by GrouchyLaTortue** [comment]
Good morning, 
Is the 'k' value the 'shape' that is mentionned in the documentation ?

**Comment by propensive** [comment]
Yes. I took the names `shape` and `scale` from [Wikipedia](https://en.wikipedia.org/wiki/Gamma_distribution) to avoid names which don't convey enough precise meaning, though `k` and `theta` are what's commonly used in statistics.
